app.js

```diff
@@ -12,6 +17,14 @@ const PORTRAIT_API = 'https://dlportraits.space/portrait_output/';
 let portraitData = null;
 let fm = null;
 
+app.use(bodyParser.json({
+  verify: (req, res, buf) => {
+    req.rawBody = buf
+  }
+}));
+
+app.use(bodyParser.json());
```

Comment:
Why two parse json middleware? Also:
```suggestion
app.use(express.json());
```